src/test/java/org/fundacionjala/sfdc/stepdefinition/ContactSteps.java

```diff
@@ -131,32 +133,39 @@ public void iClickOnOkButton() {
         contactsDetailPage.clickDeleteAlert();
 
     }
+
     /**
      * Then step.
      */
     @Then("^The system deletes the selected contact$")
     public void theSystemDeletesTheSelectedContact() {
         contactsMainPage = new SFCMainPage();
-        System.out.println("This Step tests verified id the contact:");
         Assert.assertEquals(contactsMainPage.contactHomePage(), "Home");
     }
 
     /**
      * And step.
+     *
      * @param values map.
      */
     @And("^I create a new contact$")
-    public void iCreateANewContact(final Map<String, String> values) {
+    public void iCreateANewContact(final Map<Inputs, String> values) {
         SalesForceMainTabClassic salesForceMainTab;
         salesForceMainTab = new SalesForceMainTabClassic("a.contactBlock", "li#AllTab_Tab");
         salesForceMainTab.displayOptions();
         salesForceMainTab.goToAccounts();
-
         contactsMainPage = new SFCMainPage();
         contactsMainPage.clickNewButton();
-
         contactsNewModifyPage = new SFCNewModifyPage();
-        CommonActions.setValues(values, contactsNewModifyPage.fillMethodsToFields());
+        values.keySet().stream().forEach((step) ->
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "step" parameter [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)